src/js/controllers/index.js

```diff
@@ -535,6 +535,97 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     }
   };
 
+  this.csvHistory = function() {
+
+    function saveFile(name,data) {
+      var chooser = document.querySelector(name);
+      chooser.addEventListener("change", function(evt) {
+        console.log(this.value); // get your file name
+        var fs = require('fs'); // save it now
+        fs.writeFile(this.value, data, function(err) {
+          if(err) {
+            alert("error"+err);
```

Comment:
I get that this is supposed to be a rare event but maybe we can have better error communication with the user?